en/docs/quick-start-guides/deploy-your-first-integration.md

```diff
@@ -0,0 +1,198 @@
+- Push the integration to Devant which automatically builds and deploys the integration to the development environment.
+- Test the integration.
+
+## Prerequisites
+
+1. GitHub account: [Create a GitHub account](https://github.com/signup) if you don't have one already.
+2. If you're signing in to the Devant for the first time, create an organization:
+    1. Go to [https://console.devant.dev/](https://console.devant.dev/) and sign in using your Google, GitHub, or Microsoft account.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use and Click **Create**.
+3. VSCode: [Install VSCode](https://code.visualstudio.com/download) if you don't have it installed already.
+
+## Step 1: Attach a Git repository
+1. Go to [https://console.devant.dev/](https://console.devant.dev/) and sign in. This opens the organization overview page and lists the projects.
+2. Select the **Default** project.
+3. On the project overview page, click **Attach a Git Repository**.
+4. Click **Authorize with GitHub** to connect Devant to your GitHub account. If you haven't connected your GitHub repository to Devant, authorize the WSO2 cloud app stage with your GitHub account [WSO2 Cloud App](https://github.com/marketplace/choreo-apps).
+5. Under the **Organization**  dropdown click **+ Add**. This redirects you to **Install WSO2 Cloud App Stage** page.
+6. Select your GitHub account and install [WSO2 Cloud App](https://github.com/marketplace/choreo-apps)
+
+    !!! note
+        The **WSO2 Cloud App** requires:
+        - Read and write access to code and pull requests.
+        - Read access to issues and metadata.
+
+         You can [revoke access](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-authorized-integrations#reviewing-your-authorized-github-apps) at any time. Write access is used to push changes directly to your repository.
+
+7. Select your organization under the **Organization**  dropdown. If your organization is still not listed click the Refetch button.
+8. Select a repository to save your integration. Additionally, you can select a **Branch** and a **Path** of the selected repository to save your integration.
+9. **Name** and **Identifier** fields are automatically populated. Additionally, you can edit them to your preference.
+10. Select the **Technology** as `WSO2 MI`.
+11. Choose the **Integration Type** as `Integration as API` and Click **Create**.
+
+This redirects you to the overview page of the integration. Now, let's design the integration.
+
+## Step 3: Install the MI extension
+
+1. Click **Install Micro Integrator extension** on the overview page. This opens the VSCode extensions page.
+2. Click **Install** to install the extension. This installs the development environment for MI on VSCode.
+
+[//]: # (Todo: this step may be not needed in future. MI extension will detect the Devant projct and use it)
```

Comment:
```suggestion
[//]: # (Todo: You may not need this step in the future, as the MI extension will detect and use the Devant project automatically.)
```

Reply:
Its a comment. This will be removed soon